src/cljs/rems/administration/form.cljs

```diff
@@ -74,11 +74,12 @@
    (let [id (:form/id form)]
      [:div.col.commands
       [administration/back-button "/administration/forms"]
-      [roles/when roles/show-admin-edit-buttons?
-       [edit-button id]
-       [copy-as-new-button id]
-       [status-flags/enabled-toggle form #(rf/dispatch [:rems.administration.forms/set-form-enabled %1 %2 [::enter-page id]])]
-       [status-flags/archived-toggle form #(rf/dispatch [:rems.administration.forms/set-form-archived %1 %2 [::enter-page id]])]]])
+      [roles/when roles/+admin-write-roles+
+       [:<>
```

Comment:
I think this is not an improvement. Most hiccup assumes you can just have multiple things in the body. If you want to keep `roles/when` close to Clojure `when` then I would suggest we have `roles/<>`, `roles/when-<>`, `roles/show-when` or something that keeps the `[:<>]` like before and uses the `roles/when`.

Reply:
I like show-when